packages/cacti-plugin-ledger-connector-stellar/src/test/typescript/integration/plugin-ledger-connector-stellar/run-soroban-transaction/index.test.ts

```diff
@@ -0,0 +1,359 @@
+import { AddressInfo } from "net";
+import { pluginName } from "..";
+import {
+  StellarApiClient,
+  StellarApiClientOptions,
+} from "../../../../../main/typescript/api-client/stellar-api-client";
+import {
+  IListenOptions,
+  LogLevelDesc,
+  Servers,
+} from "@hyperledger/cactus-common";
+import { PluginRegistry } from "@hyperledger/cactus-core";
+import { PluginFactoryLedgerConnector } from "../../../../../main/typescript/plugin-factory-ledger-connector";
+import { loadWasmFile } from "../../../../../main/typescript/utils";
+import { StellarTestLedger } from "@hyperledger/cactus-test-tooling";
+import { NetworkConfig } from "stellar-plus/lib/stellar-plus/network";
+import { PluginLedgerConnectorStellar } from "../../../../../main/typescript/plugin-ledger-connector-stellar";
+import http from "http";
+import { Network } from "stellar-plus/lib/stellar-plus";
+import { Constants, PluginImportType } from "@hyperledger/cactus-core-api";
+import { v4 as uuidV4 } from "uuid";
+import express from "express";
+import { Server as SocketIoServer } from "socket.io";
+import bodyParser from "body-parser";
+import { DefaultAccountHandler } from "stellar-plus/lib/stellar-plus/account";
+import { K_CACTUS_STELLAR_TOTAL_TX_COUNT } from "../../../../../main/typescript/prometheus-exporter/metrics";
+
+const testCaseName = pluginName + " / run soroban transactions";
+const runSorobanTransactionFnTag = `PluginLedgerConnectorStellar#invokeContract()`;
+
+describe(testCaseName, () => {
+  const logLevel: LogLevelDesc = "TRACE";
+  const stellarTestLedger = new StellarTestLedger({ logLevel });
+  let networkConfig: NetworkConfig;
+  let wasmBuffer: Buffer;
+  let connector: PluginLedgerConnectorStellar;
+  let server: http.Server;
+  let apiClient: StellarApiClient;
+  const contractIdPattern = /^C[A-Z0-9]{55}$/;
+  const wasmHashPattern = /^[a-f0-9]{64}$/;
+
+  // spell-checker:disable
+  const tokenSpec = [
```

Comment:
@fazzatti Please reference the source code that these specs were generated from

Reply:
Thanks @petermetz!
I've moved this to the smart contract project with instructions in the readme file so other people can consume it properly and, also generate their own whenever working on their custom projects.